pyproject.toml

```
@@ -0,0 +1,15 @@
+[project]
+name = "g-adopt-github-io"
+version = "0.1.0"
+description = "The website and documentation for the g-adopt project"
+readme = "README.md"
+requires-python = ">=3.11"
```

```suggestion
requires-python = ">=3.10"
```
I've confirmed this builds and runs fine on Python 3.10, and this change fixes the `uv.lock` changes problem. I think I see the value in `uv.lock` now, having full set of dependencies anyone could have all there makes it easier to find what might have caused a breaking change.

---

Curious, I was hitting a weird incompatibility with 3.10 which is why I bumped the version in the first place...

---

How strange...
```
[dale@DESKTOP-93BIFKT g-adopt.github.io]$ uv run -- mkdocs serve
Using CPython 3.10.16 interpreter at: /opt/python3/3.10.16/bin/python3
Creating virtual environment at: .venv
Installed 85 packages in 160ms
INFO    -  DeprecationWarning: Jupyter is migrating its paths to use standard platformdirs
           given by the platformdirs library.  To remove this warning and
           see the appropriate new directories, set the environment variable
           `JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
           The use of platformdirs will be the default in `jupyter_core` v6
             File
           "/home/dale/g-adopt.github.io/.venv/lib/python3.10/site-packages/jupyter_core/utils/__init__.py",
           line 90, in deprecation
               warnings.warn(message, DeprecationWarning, stacklevel=stacklevel + 1)
             File "/home/dale/g-adopt.github.io/.venv/lib/python3.10/site-packages/jupyter_client/connect.py",
           line 22, in
               from jupyter_core.paths import jupyter_data_dir, jupyter_runtime_dir, secure_write
INFO    -  Building documentation...
INFO    -  FutureWarning: The 'import' key is renamed 'inventories' for the Python handler
             File
           "/home/dale/g-adopt.github.io/.venv/lib/python3.10/site-packages/mkdocstrings/handlers/base.py",
           line 682, in get_handler
               self._handlers[name] = module.get_handler(**kwargs)
             File
           "/home/dale/g-adopt.github.io/.venv/lib/python3.10/site-packages/mkdocstrings_handlers/python/handler.py",
           line 370, in get_handler
               warn("The 'import' key is renamed 'inventories' for the Python handler", FutureWarning,
           stacklevel=1)
INFO    -  Cleaning site directory
INFO    -  Documentation built in 0.46 seconds
INFO    -  [18:03:36] Watching paths for changes: 'docs', 'mkdocs.yml'
INFO    -  [18:03:36] Serving on http://127.0.0.1:8000/
```

---

:exploding_head:

---

I think my computer is just silly.